src/components/Character/styles.scss

```diff
@@ -1,19 +1,147 @@
 .wrapper {
-	font-size: 1em;
+	position: absolute;
+	top: 0;
+	width: 100%;
+	height: 100%;
 
 	&[aria-hidden="true"]{
 		opacity: 0;
+		pointer-events: none
 	}
 }
-	.button {
-		&.next {
-			font-size: 1em;
+
+	.container {
+		display: flex;
+		width: 100%;
+		height: 100%;
+		flex-direction: column;
+		justify-content: flex-end;
+		align-items: center;
+		font-size: 1.25rem;
+		--padding_container: 10vw;
+	}
+
+		.content {
+			position: relative;
+			width: 75%;
+			height: 50%;
+			z-index: 1;
 		}
-		&.previous {
-			font-size: 1em;
+	
+			.text {
+				overflow-y: scroll;
+			}
+
+			.name {
+				margin-bottom: 0.4em;
+				font-size: 5em;
+				font-family: var(--font_family__heading);
+			}
+
+			.nickName {
+				margin-bottom: 1.35em;
+				font-size: 2.6em;
+				font-family: var(--font_family__heading);
+			}
+
+			.actorName {
+				margin-bottom: 1.7em;
+				font-size: 1em;
+				font-family: var(--font_family__body);
+				text-transform: uppercase;
+			}
+
+			.bio {
+				margin-bottom: 2.4em;
+				font-size: 1em;
+				line-height: 1.5;
+				font-family: var(--font_family__body);
+			}
+
+		.hero {
+			position: fixed;
+			width: 100%;
+			height: 100%;
+			top: 0;
+			left: 0;
+			z-index: 0;
+			background-color: crimson;
 		}
+
+		.controls {
+			display: flex;
+			width: 100%;
+			justify-content: space-between;
+			font-size: 1.1em;
+			font-family: var(--font_family__heading);
+		}
+
+			.button {
+				display: flex;
+				align-items: center;
+				text-decoration: none;
+				--button_margin: 1.2em;
+
+				&::before, 
+				&::after {
+					display: inline-block;
+					font-size: 1.36em;
+					width: 1em;
+					height: 1.5em;
+					background-color: lime;
+				}
+			}
+
+			.previous {
+				left: var(--padding_container);
+
+				&::before {
+					content: "";
+					margin-right: var(--button_margin);
```

Comment:
```suggestion
```
Whatever this is doing is definitely something it shouldn't - kill these and find another way, haha. I think moving the `.controls` inside your `.content` is going to solve most of your problems here.